src/hub/tests/unit/stripe/utils.py

```diff
@@ -28,10 +28,10 @@ def run_event_process(event) -> Response:
 
 
 class MockSqsClient:
-    def list_queues(QueueNamePrefix={}) -> Any:  # type: ignore
+    def list_queues(QueueNamePrefix=None) -> Any:  # type: ignore
```

Comment:
i think you need to check if QueueNamePrefix is None and if so set to a default value; you have to keep the = None part though; same for the rest

Reply:
It's actually not used at all presently.